nextcord/channel.py

```diff
@@ -782,7 +789,455 @@ def archived_threads(
             The archived threads.
         """
         return ArchivedThreadIterator(self.id, self.guild, limit=limit, joined=joined, private=private, before=before)
+    
+    
+class ForumChannel(abc.GuildChannel, Hashable):
+    # TODO: docstring
+    """Represents a Discord guild forum channel.
 
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two channels are equal.
+            
+        .. describe:: x != y
+
+            Checks if two channels are not equal.
+
+        .. describe:: hash(x)
+
+            Returns the channel's hash.
+
+        .. describe:: str(x)
+
+            Returns the channel's name.
+    """
+    
+    __slots__ = (
+        'id',
+        'guild',
+        'name',
+        'category_id',
+        'position',
+        'topic',
+        'nsfw',
+        'slowmode_delay',
+        'default_auto_archive_duration',
+        'last_message_id',
+        '_state',
+        '_type',
+        '_overwrites',
+    )
+    
+    def __init__(self, *, state: ConnectionState, guild: Guild, data: ForumChannelPayload):
+        self._state: ConnectionState = state
+        self.id: int = int(data['id'])
+        self._type: int = data['type']
+        self._update(guild, data)
+        
+    def _update(self, guild: Guild, data: ForumChannelPayload) -> None:
+        self.guild = guild
+        self.name = data['name']
+        self.category_id: Optional[int] = utils._get_as_snowflake(data, 'parent_id')
+        self.topic: Optional[str] = data.get('topic')
+        self.position: int = data['position']
+        self.nsfw: bool = data.get('nsfw', False)
+        # Does this need coercion into `int`? No idea yet. 
+        self.slowmode_delay: int = data.get('rate_limit_per_user', 0)
+        self.default_auto_archive_duration: ThreadArchiveDuration = data.get('default_auto_archive_duration', 1440)
+        self.last_message_id: Optional[int] = utils._get_as_snowflake(data, 'last_message_id')
+        self._fill_overwrites(data)
+        
+    async def _get_channel(self):
```

Comment:
```python
async def _get_channel(self) -> "ForumChannel":

Reply:
```suggestion
    async def _get_channel(self) -> ForumChannel:
```
better

Reply:
> better

This is invalid code, if you had tested that you would know. 

Please don't contradict suggested code when you don't understand what your talking about.